tests/ts_A_create_table_not_null_pk_fields.erl

```diff
@@ -0,0 +1,37 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ts_A_create_table_not_null_pk_fields).
+
+-behavior(riak_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-export([
+         confirm/0
+        ]).
+
+confirm() ->
+    ClusterType = single,
+    DDL = ts_util:get_ddl(not_null_primary_key_field_fail),
```

Comment:
@hmmr Not sure who designed this originally (Gordon?), but I'm quite happy with the the SQL being with the tests as @andytill suggests.  As he also said, the advantage to having (some) of the basic SQL/DDL in `ts_utils` is that we don't have lots of duplication and updates can be made in one place.  So if it's used in multiple places, let's keep it  in `ts_utils` for now, but move unique SQL/DDL to the respective tests.